nbactions-dev.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Probably it should be removed ?

Reply:
Netbeans isn't my preferred IDE - removed